picard/track.py

```diff
@@ -235,6 +235,13 @@ def can_edit_tags(self):
     def can_view_info(self):
         return self.num_linked_files == 1 or self.metadata.images
 
+    def can_extract(self):
+        """Return if this object has a linked file with a recordingId, required by the AcosuticBrainz feature extractor."""
+        for file in self.files:
+            if file.metadata["musicbrainz_recordingid"]:
```

Comment:
Maybe `if file.can_extract():` ?

Reply:
Yes. Leftover from when I assumed only files associated with tracks would get extracted, which is true, so I didn't include the file.can_extract. =x